.travis.yml

```diff
@@ -12,12 +12,8 @@ before_install:
   - npm config set ca ""
 install:
   - curl -L https://rawgit.com/marionettejs/travis-ci-setup/master/ci.sh | bash
+  - npm install travis-ci
 after_success:
   - if [[ $MAINRUN == true ]]; then npm run-script coveralls; fi
-before_deploy:
-  - npm install travis-ci
-deploy:
```

Comment:
I'm not familiar with Travis configs. Why does the `deploy` step need to be removed?